test/spec/crud/unified/bulkWrite-updateMany-let.yml

```diff
@@ -85,13 +85,12 @@ tests:
                   - q: *filter
                     u: *update
                     multi: true
-                    upsert:
-                      $$unsetOrMatches: false
+                    upsert: { $$unsetOrMatches: false }
                 let: *let
 
     outcome:
       - collectionName: *collection0Name
         databaseName: *database0Name
         documents:
           - { _id: 1, x: 20 }
-          - { _id: 2, x: 21 }
+          - { _id: 2, x: 21 }
```

Comment:
I'm leaving one comment because I didn't want to leave comments everywhere this occurred.

There are a lot of removed newlines at the ends of files in this PR that aren't included in the actual spec changes (from [here](https://github.com/mongodb/specifications/commit/51ca03b07d0c6681951d82043aa5b49869a66e87)).  Can we add these newlines back?  This will keep the changes in this PR consistent with the changes in the specs repo and make future changes to these files simpler.

Reply:
oh ok yeah i think they occurred when i copied the file contents over. I'll go ahead and add them now!